packages/st2/debian/control

```diff
@@ -4,16 +4,16 @@ Priority: optional
 Maintainer: StackStorm Engineering <opsadmin@stackstorm.com>
 Build-Depends: debhelper (>= 9),
                python,
-               dh-virtualenv (>= 0.8)
+               dh-virtualenv (>= 0.8),
 Standards-Version: 3.9.5
 Homepage: http://stackstorm.com/
 Vcs-Git: git://github.com/stackstorm/st2.git
 Vcs-Browser: https://github.com/stackstorm/st2
 
 Package: st2
 Architecture: any
-Pre-Depends: dpkg (>= 1.16.16), python2.7, ${misc:Pre-Depends}, adduser
```

Comment:
I wouldn't touch such things like `${misc:Pre-Depends}` as it's something that's autogenerated by the debhelper based on dependencies he'll find package relies on.

Reply:
Yeah, I was just testing it - I will put it back once I confirm everything is working.